As in Odoo, the existence of an ``__init__.py`` file is now asserted to determine valid addons path